config/locales/idv/es.yml

```diff
@@ -32,6 +32,11 @@ es:
               su cuenta, no habrá verificado su identidad.
             - No obstante, continuará teniendo una cuenta en %{app_name}. Puede
               administrar o eliminar su cuenta desde %{account_page_text}.
+        gpo:
+          start_over: Si borras tu información y empiezas de nuevo
+          warnings:
+            - El código único de tu mensaje ya no funcionará
```

Comment:
'mensaje' is message. Do we want 'carta' (letter) here? English and French say letter. @artfulaction

Reply:
Good catch, I see "carta" elsewhere to refer tho this, so I'm good with making this change. Thanks @soniaconnolly !

Reply:
I'm rereading this now, and it should also say 'su' rather than 'tu' (informal) in both `start_over` and `warnings`.